### PR TITLE
Bump version for lix sample

### PR DESCRIPTION
**Pull request by timsneath**
Current version is incompatible with what's on the Flutter `master` channel.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by timsneath** [comment]
Holding back a moment, since @aam commented on another issue that suggested this SDK constraint might be too broad. Waiting for a consistent verdict here that we can then make sure we apply across the board.

**Comment by mit-mit** [comment]
@timsneath thanks for updating this! I believe the correct upper constraint (as discussed e.g. in https://github.com/dart-lang/sdk/issues/43757#issuecomment-707371249) is to use an upper constraint of `<2.12.0` to express that until the feature is stable we should not make claims about forward compatibility.

File: null_safety/calculate_lix/pubspec.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: calculate_lix
 description: A command-line application demonstrating use of null safety.
-version: 1.1.0
+version: 1.1.1
 
 environment:
-  sdk: '>=2.10.0-56.0.dev <2.11.0'
+  sdk: '>=2.10.0-56.0.dev <3.0.0'
```
Comment (mit-mit) [inline null_safety/calculate_lix/pubspec.yaml:6]: Should use `<2.12.0`

**Review by mit-mit — CHANGES_REQUESTED** [review]

**Comment by mit-mit** [comment]
It would also be great to add something like this to `/calculate_lix/README.md`:

```
--- a/null_safety/calculate_lix/README.md
+++ b/null_safety/calculate_lix/README.md
@@ -41,6 +41,10 @@ To run the main app, type these commands in the terminal/command-prompt:
     - `/Users/michael/dev/preview/dart-sdk/bin/dart pub get`
     - `/Users/michael/dev/preview/dart-sdk/bin/dart --enable-experiment=non-nullable run bin/main.dart text/lorem-ipsum.txt`

+*Note*: You may see a warning "Overriding the upper bound Dart SDK constraint"
+when running pub get. You can ignore this, which is a temporary issue only
+expected during preview.
+
 ### Running from VSCode
```

**Review by mit-mit — APPROVED** [review]

**Pull request merged**